src/tests/inbound/createInbound/downloadDocsFromSendPage.test.ts

```diff
@@ -0,0 +1,235 @@
+      await createInboundPage.sendStep.downloadButton.click();
+      await expect(
+        createInboundPage.sendStep.getDocuments('Export Packing List (.xls)')
+      ).toBeVisible();
+      await expect(
+        createInboundPage.sendStep.getDocuments('Packing List')
+      ).toBeVisible();
+      await expect(
+        createInboundPage.sendStep.getDocuments('Certificate of Donation')
+      ).toBeVisible();
+      await expect(
+        createInboundPage.sendStep.getDocuments('Delivery Note')
+      ).not.toBeVisible();
+    });
+
+    const certificateOfDonationFileName =
+      `Certificate of Donation - ` +
+      `${inboundShipmentIdentifier}`.toString().trim() +
+      `.xls`;
+    const exportPackingListFileName =
+      `Shipment ` +
+      `${inboundShipmentIdentifier}`.toString().trim() +
+      ` - Packing List.xls`;
+    const packingListFileName =
+      `Packing List - ` +
+      `${ORIGIN.locationNumber}` +
+      `-` +
+      `${CURRENT_LOCATION.name}-` +
+      `${formatDate(TODAY, 'DDMMMYYYY')}-` +
+      `${DESCRIPTION}`.replace(' ', '') +
+      `.xls`;
+    //assertion on packingListFileName might not be stable always and it's based on data so might require adjustments
+    //if origin doesn't have locationNumber it would use origin name
+    //it might also fail when locations have very long names
+
+    await test.step('Download Certificate of Donation file', async () => {
+      const popupPromise = page.waitForEvent('popup');
+      await createInboundPage.sendStep
+        .getDocuments('Certificate of Donation')
+        .click();
+      const popup = await popupPromise;
+      const downloadPromise = popup.waitForEvent('download');
+      const download = await downloadPromise;
+      await popup.close();
+      await expect(download.suggestedFilename()).toBe(
+        certificateOfDonationFileName
+      );
+    });
+
+    await test.step('Download Export Packing List (.xls) file', async () => {
+      await createInboundPage.sendStep.isLoaded();
+      const popupPromise = page.waitForEvent('popup');
+      await createInboundPage.sendStep
+        .getDocuments('Export Packing List (.xls)')
+        .click();
+      const popup = await popupPromise;
+      const downloadPromise = popup.waitForEvent('download');
+      const download = await downloadPromise;
+      await popup.close();
+      await expect(download.suggestedFilename()).toBe(
+        exportPackingListFileName
+      );
+    });
+
+    await test.step('Download Packing list file', async () => {
+      await createInboundPage.sendStep.isLoaded();
+      const popupPromise = page.waitForEvent('popup');
+      await createInboundPage.sendStep.getDocuments('Packing List').click();
+      const popup = await popupPromise;
+      const downloadPromise = popup.waitForEvent('download');
+      const download = await downloadPromise;
+      await popup.close();
+      await expect(download.suggestedFilename()).toBe(packingListFileName);
```

Comment:
because it's not stable it's good to know which part of the file name can change. For now, I think it can be something like:
```suggestion
      await expect(download.suggestedFilename()).toMatchText(/^Packing List - .*\.xls(x)?$/);
```
It will match file names starting with 'Packing List - ' and ending with .xls or .xlsx